docs/api-docs/in-depth-guides/rail-integration-guide.mdx

```diff
@@ -1,100 +1,154 @@
+For a broader overview, including the reasons why you'd want rail visibility and how to use it in the Terminal49 dashboard,
+ [read our announcement post](https://www.terminal49.com/blog/launching-north-american-intermodal-rail-visibility-on-terminal49/).
+
+## Table of Contents
+
+- [Supported Rail Carriers](#supported-rail-carriers)
+- [Supported Rail Milestones and Data Attributes](#supported-rail-milestones-and-data-attributes)
+  - [Rail Milestones (aka rail events)](#rail-milestones-aka-rail-events)
+  - [Webhook Notifications](#webhook-notifications)
+  - [Rail Container Attributes](#rail-container-attributes)
+- [Integration Methods](#integration-methods)
+  - [Integration via API](#a-integration-via-api)
+  - [Integration via DataSync](#b-integration-via-datasync)
+
+## Supported Rail Carriers
+
+Terminal49 container tracking platform integrates with all North American Class-1 railroads that handle container shipping, providing comprehensive visibility into your rail container movements.
 
-### Supported Rail Carriers
 - BNSF Railway
 - Canadian National Railway (CN)
 - Canadian Pacific Railway (CP)
 - CSX Transportation
-- Kansas City Southern Railway (KCS)
 - Norfolk Southern Railway (NS)
 - Union Pacific Railroad (UP)
 
 By integrating with these carriers, Terminal49 ensures that you have real-time access to critical tracking data, enabling better decision-making and operational efficiency.
 
-### Supported Rail Milestones and Data Attributes
+## Supported Rail Milestones and Data Attributes
+
+Terminal49 seamlessly tracks your containers as they go from container ship, to ocean terminal, to rail carrier.
+
+We provide a set of milestones/events that let you track the status of your containers as they move through the rail system.  You can be notified by webhook whenever these events occur.
+
+We also provide a set of attributes that let you know the current status of your container at any given time, as well as useful information such as ETA, pickup facility, and availability information.
 
-Terminal49 tracks various milestones and data attributes that are essential for effective rail container management. The key events and attributes include:
+### Rail Milestones (aka rail events)
 
+{/* TODO: create rail-carriers page or section */}
+There are several core milestones that occur on most rail journeys.  Some rail carriers do not share all events, {/* (see [Rail Carriers](#rail-carriers) for more details), */} but in general these are the key events for a container.
 
 ```mermaid
 graph LR
+A[Rail Loaded] --> B[Rail Departed]
+B --> C[Rail Arrived]
+C --> D[Rail Unloaded]
+D --> E[Full Out]
+E --> F[Empty Return]
+```
+
+However, not all shipments have a simple journey.  You may want to track events such as when a train passes through a town, when the container switches trains or rail carriers at an interchange, and when the status of your container at the terminal changes.
```

Comment:
I don't think this is helpful unless we expose these events through the api

Reply:
Removed, but replaced with another diagram that zooms in on the rail-terminal statuses